src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+import java.util.Scanner;
+import duke.task.*;
+import duke.exceptions.*;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    public static void main(String[] args) throws FileNotFoundException, IOException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        try {
+            printFileContents("duke.txt");
```

Comment:
You could declare a filepath constant variable for the file name which is being reused.